### PR TITLE
Eai 64

### PR DESCRIPTION
**Pull request by ntlong1594**
1. Add new column id_service_integrated: id_service_integrated, VARCHAR(30), required = true

2. Make id_user_integrated column as required = false

### CODE REVIEW

File: src/application/models/v3/Patient_model.php [hunk 1]
```diff
@@ -19,9 +19,9 @@ public function add($patient_integrated)
     public function validate($patient_integrated) {
         $this->load->helper('data_validation');
 
-        if (empty($patient_integrated['id_user_integrated']))
```
Comment (jasminecrazy) [inline src/application/models/v3/Patient_model.php:22]: Why did you remove this validation

**Comment by jasminecrazy** [comment]
@ntlong1594 Why I didn't see any update for sql file. Could you show me which commit contain 
`Add new column id_service_integrated: id_service_integrated, VARCHAR(30), required = true`

File: src/application/models/v3/Patient_model.php (hunk 1 above)
Comment (ntlong1594) [reply src/application/models/v3/Patient_model.php:22]: Make id_user_integrated column as required = false , following ticket requirement

**Comment by ntlong1594** [comment]
> @ntlong1594 Why I didn't see any update for sql file. Could you show me which commit contain
> `Add new column id_service_integrated: id_service_integrated, VARCHAR(30), required = true`

added

**Review by chicanh — APPROVED** [review]
